Tests/ApodiniTests/PropertiesTests/BindingTests.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  BindingTests.swift
+//  
+//
+//  Created by Max Obermeier on 24.02.21.
+//
+
+import Foundation
+
+@testable import Apodini
+import XCTest
+import XCTApodini
+import ApodiniREST
+
+final class BindingTests: ApodiniTests, EnvironmentAccessible {
+    struct Greeter: Handler {
+        @Binding var country: String?
+        
+        func handle() -> String {
+            country ?? "World"
+        }
+    }
+    
+    struct ReallyOptionalGreeter: Handler {
+        @Binding var country: String??
+        
+        func handle() -> String? {
+            if let country = self.country {
+                return country ?? "World"
+            } else {
+                return nil
+            }
+        }
+    }
+        
+    var featured: String? = "Italy"
+    
+    @PathParameter var selectedCountry: String
+    @Environment(\BindingTests.featured) var featuredCountry
+    
+    @ComponentBuilder
+    var testService: some Component {
+        Greeter(country: .constant(nil))
```

Comment:
As commented above with the following extension
```swift
extension Binding: ExpressibleByNilLiteral where Value: ExpressibleByNilLiteral {
    public init(nilLiteral: ()) {
        self.init(constant: nil)
    }
}
```
allows you to write:
```suggestion
        Greeter(country: nil)
```

Reply:
Done with 5195cbc354c8f508b37339215035bdad1e4997ab